apps/remix-ide-e2e/src/tests/plugin_api.ts

```diff
@@ -112,6 +112,7 @@ const clickAndCheckLog = async (browser: NightwatchBrowser, buttonText: string,
   if (methodResult && typeof methodResult !== 'string') { methodResult = JSON.stringify(methodResult) }
   if (eventResult && typeof eventResult !== 'string') { eventResult = JSON.stringify(eventResult) }
   if (buttonText) {
+    console.log('this is buttonText in this test', buttonText)
```

Comment:
Is this log needed? If not, please remove.

Reply:
sorry left this while working here.